WebContent/WEB-INF/jsp/collectionStatusReport.jsp

```diff
@@ -0,0 +1,245 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"%>
+<%@ page errorPage="/jsp/errorPage.jsp"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+
+<c:set var="tsNameLen"> ${fn:length(model.triplestore)}</c:set>  
+<c:set var="tsNameFl" scope="page">${fn:substring(model.triplestore, 0, 1)}</c:set> 
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+<jsp:include flush="true" page="/jsp/libheader.jsp" />
+<script src="//code.jquery.com/jquery-1.10.2.js"></script>
+<script src="//code.jquery.com/ui/1.11.1/jquery-ui.js"></script>
+<style>
+.i_group { width: 320px; }
+</style>
+</head>
+<body style="background-color:#fff;">
+<script type="text/javascript">
+    function confirmSubmit() {
+       // begin / end date validation
+       var beginDate = document.mainForm.beginDate.value.trim();
+
+       if (beginDate.length == 0) {
+           alert ("Please enter begin date in format yyyy-mm-dd.");
+           document.mainForm.beginDate.focus();
+           return false;
+       }
+
+       beginDate = parseDate(beginDate);
+       if (beginDate == null) {
+           alert ("Please enter a valid begin date in format yyyy-mm-dd.");
+           document.mainForm.beginDate.focus();
+           return false;
+       }
+
+        if (endDate.length > 0) {
+            endDate = parseDate(endDate);
+            if (endDate == null) {
+                alert ("Please enter a valid end date in format yyyy-mm-dd.");
+                document.mainForm.endDate.focus();
+                return false;
+            }
+        }
+
+        if (beginDate != null && endDate != null) {
+            if (beginDate > endDate) {
+                alert ("Invalid date range: end date must NOT before the begin date!");
```

Comment:
Minor typo. I think you want `must NOT be before`? I think the `be` is what's missing.